eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -163,7 +164,10 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
         maxAcceptedHtlcs = localParams.maxAcceptedHtlcs,
         fundingPubkey = fundingPubKey,
         revocationBasepoint = keyManager.revocationPoint(channelKeyPath).publicKey,
-        paymentBasepoint = keyManager.paymentPoint(channelKeyPath).publicKey,
+        paymentBasepoint = channelVersion match {
+          case v if v.isSet(USE_STATIC_REMOTEKEY_BIT) => localParams.localPaymentBasepoint.get
+          case STANDARD => keyManager.paymentPoint(channelKeyPath).publicKey
+        },
```

Comment:
Good catch, i pushed the fix in cd4149d but using the wildcard instead of `if .. else ..`, it felt more consistent with other part of the code where is used the pattern matching